src/Mono.Android/Java.Lang/Object.cs

```diff
@@ -92,31 +59,26 @@ internal void SetHandleOnDeserialized (StreamingContext context)
 			JNIEnv.FinishCreateInstance (Handle, "()V");
 		}
 
-#if JAVA_INTEROP
 		[EditorBrowsable (EditorBrowsableState.Never)]
-		public int JniIdentityHashCode {
-			get {return (int) key_handle;}
-		}
+		public int JniIdentityHashCode => base.JniIdentityHashCode;
 
 		[DebuggerBrowsable (DebuggerBrowsableState.Never)]
 		[EditorBrowsable (EditorBrowsableState.Never)]
-		public JniObjectReference PeerReference {
-			get {
-				return new JniObjectReference (handle, (JniObjectReferenceType) handle_type);
-			}
-		}
+		public JniObjectReference PeerReference => base.PeerReference;
```

Comment:
Does this need the `new` keyword to avoid a C# warning?
* https://github.com/dotnet/java-interop/blob/c86ae26c5814766af6e1a191d94ac29c0eb6039b/src/Java.Interop/Java.Interop/JavaObject.cs#L38